applications/portal/backend/api/tests/test_antibodies.py

```diff
@@ -108,14 +109,14 @@ def test_create(self):
         a.save()
 
         duplicated = AddAntibodyDTO(**example_ab)
-        # duplicated.vendorName = "My vendor synonym" # should keep the same vendor and add a synonym
-        da = create_antibody(duplicated, "bbb")
-
-        assert da.accession != da.abId
```

Comment:
Removing this because this is generated new with  self._generate_ab_id() and won't be same 

![image](https://github.com/MetaCell/scicrunch-antibody-registry/assets/59233227/fc658116-05e5-4b47-8b9f-9efa208255af)